package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "build": "yarn run build:ts && yarn run gen:locale && yarn run build:extension",
     "gen:locale": "yarn run ts-node scripts/generateLocale.ts",
-    "build:ts": "yarn run clean:ts && rollup -c --failAfterWarnings && sed -i '/setTimeout/d' dist/thirdparty/prismjs.js && sed -i 's/var\\ /let\\ /g' dist/extension.js dist/prefs.js",
+    "build:ts": "yarn run clean:ts; rollup -c --failAfterWarnings && sed -i '/setTimeout/d' dist/thirdparty/prismjs.js && sed -i 's/var\\ /let\\ /g' dist/extension.js dist/prefs.js",
```

Comment:
Why was this changed from  `&&` to `;` did `yarn run clean:ts` fail locally? since it would be better, if the rest only gets executed, if that also works.